mapDownload.properties

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
yes, that one won't be active until this is actually merged. The third config for local file 'triplea_maps.xml' should work.